### PR TITLE
Hearing Prep Readme

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #2824

### CODE REVIEW

File: docs/hearing-prep.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+1. Select Upcoming Days.
+
+## Supported Browsers
+
+- Chrome
+
+## Supported Screen Size
+
+- 1280 x 800 resolution
```
Comment (nolawi) [inline docs/hearing-prep.md]: should we add `minimum` supported screen resolution.. ?

File: docs/hearing-prep.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# What is Caseflow Hearing Prep?
+
+Caseflow Hearing Prep is a web application that allows judges to prepare for hearings more quickly. It pulls relevant information in from BGS and VACOLS and provides a worksheet for the judge to fill out before each hearing.
+
```
Comment (laurjpeterson) [inline docs/hearing-prep.md]: Thinking it's worth listing a little bit more of the judge's flow here. How about the statement below? I like the BGS and VACOLS line, so feel free to integrate that into this below statement however it flows.

It allows judges to view their upcoming hearings, see a list of hearings for a given hearing day, view pertinent data about an appellant's case, and provides a Hearing Worksheet for judges' notes during prep and during a hearing. Also, it allows judges to access Reader from Hearing Prep so they can review the Veteran's eFolder while preparing for a hearing.

Comment (annienguyenUSDS) [reply docs/hearing-prep.md]: Tried to combine and add in a few more things to What is Caseflow. 

Caseflow Hearing Prep is a web application that allows judges to prepare for hearings more quickly and efficiently. It allows judges to view their upcoming hearings, see a list of hearings for a given hearing day, view pertinent data about an appellant's case, and provide a Hearing Worksheet for the judge to fill out before and during prep by pulling in relevant information from BGS and VACOLS. It also allows judges access to Reader from Hearing Prep so they can readily review the Veteran's documents while preparing and taking notes during a hearing.

File: docs/hearing-prep.md (hunk 1 above)
Comment (sharonwarner) [reply docs/hearing-prep.md]: Yeah, @lakohl @annienguyenUSDS, do either of you know what the minimum supported screen resolution is?

**Comment by sharonwarner** [comment]
Moving to appeals-pm repo

**Pull request closed without merging**